.gitignore

```diff
@@ -16,9 +16,9 @@ coverage
 _public
 bundle-analyzer
 
-cypress/videos/
-cypress/screenshots/
-cypress/downloads/
+**/cypress/videos/
```

Comment:
Correct me if I'm wrong. I just remembered that changing `cypress/videos/` to `**/cypress/videos/` seems identical to `git-ignore`

Reply:
My bad 🥹